src/model/errors.rs

```diff
@@ -0,0 +1,21 @@
+use std::fmt;
+
+#[derive(Debug)]
+pub struct RobotparserError {
+    pub kind: ErrorKind,
+}
+
+#[derive(Debug)]
+pub enum ErrorKind {
+    Url(url::ParseError),
+    HttpClient(reqwest::Error),
```

Comment:
`HttpClient` => `Http`

Reply:
Done in df30fe6701ba4b6df25660ded4d0161026cc37f6